pkgs/by-name/vl/vlc/package.nix

```diff
@@ -253,7 +253,7 @@ stdenv.mkDerivation (finalAttrs: {
     # Upstream luac can't cross compile, so we have to install the lua sources
     # instead of bytecode:
     # https://www.lua.org/wshop13/Jericke.pdf#page=39
-    + lib.optionalString (!stdenv.hostPlatform.canExecute stdenv.buildPlatform) ''
+    + lib.optionalString (!stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
```

Comment:
I think this is correct.  It's been a long time, but what I guess happens here is that since luac can't cross compile, it always compiles code for the platform it's being run on, the build platform.  If the host platform can execute code for the build platform, then we can just install that compiled code and it's fine.  If it can't, we have to fall back to interpreting.